src/cli-controller.js

```diff
@@ -10,6 +10,10 @@ import {
 } from './index';
 
 function normalizePath(filename) {
```

Comment:
Can this whole function just be replaced with `path.resolve`?

ref: https://nodejs.org/api/path.html#path_path_resolve_paths